.github/workflows/release-tag-version.yml

```diff
@@ -0,0 +1,69 @@
+name: release-tag-version
+on:
+  push:
+    tags:
+      - 'v*'
+
+jobs:
+  build-and-release:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 2
+
+      - uses: actions/setup-go@v4
+        with:
+          go-version: '1.18.1'
+
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 11.11.0
+          cache: yarn
+          cache-dependency-path: ui/yarn.lock
+
+      - run: mkdir -p ~/bin/ && export PATH="~/bin/:$PATH"
+
+      - name: Install UPX
+        uses: crazy-max/ghaction-upx@v3
+        with:
+          install-only: true
+          version: v3.95
+
+      - run: make download-tools
+
+      - run: docker version
+
+      - run: make install
+
+      - run: make generate
+      - run: make build-js
+
+      - uses: docker/setup-qemu-action@v3
+      - uses: docker/setup-buildx-action@v3
+
+      - uses: docker/metadata-action@v5
+        id: meta
+        with:
+          images: traggo/server
+          tags: |
+            type=semver,pattern={{version}}
+
+      - uses: docker/login-action@v3
+        with:
+          username: ${{ secrets.CONTAINER_REGISTRY_USER }}
+          password: ${{ secrets.CONTAINER_REGISTRY_PASSWORD }}
+
+      - uses: docker/build-push-action@v5
```

Comment:
This probably doesn't work because the current docker/Dockerfile doesn't build the binary. I can add this to my to-do list, but it may have to wait until the end of next week.

Reply:
No rush. I was just trying to give you a bit of a head start since I have a little experience with Actions.